docs/intro/tutorial02.txt

```diff
@@ -488,10 +488,10 @@ system.
 Customizing your *project's* templates
 --------------------------------------
 
-Create a ``templates`` directory in your project directory. Templates can
+Create a ``templates`` directory under your project directory. Templates can
 live anywhere on your filesystem that Django can access. (Django runs as
-whatever user your server runs.) However, keeping your templates within the
-project is a good convention to follow.
+whatever user your server runs.) However, placing your templates folder
+next to your project settings file is a good convention to follow.
```

Comment:
Hi,
Doesn't DIRS (below) point to a templates folder next to your project _manage.py_ file? (not settings.py file)
-Collin